src/js/player.js

```diff
@@ -3305,7 +3305,7 @@ class Player extends Component {
   handleSrc_(source, isRetry) {
     // getter usage
     if (typeof source === 'undefined') {
-      return this.cache_.src || '';
```

Comment:
I would be careful with this change. I don't remember all the details, but I believe `src` and `source` are specifically defined to be allowed to be different in some cases. I believe that maybe contrib-ads relies on that?